1.0/docs/devguide/styling.md

```diff
@@ -699,3 +699,21 @@ that includes a `custom-style` element.
       } 
     </style>      
 
+## Third-party libraries that modify local DOM
+
+If you are using a third-party library that adds local DOM nodes to your
+Polymer element, you may notice that styles on the element do not update
+properly. 
+
+The correct way to add DOM nodes to a Polymer element's local DOM is via
```

Comment:
Description is close but not quite right—we usually talk about the Polymer DOM API (rather than the `Polymer.dom` method) because the API structure is kind of weird. Briefly, the `Polymer.dom(node)` returns an unnamed object that lets you manipulate `node` in a way that respects local DOM. I think I've called that object a wrapper elsewhere in the docs, although it's not clear to me whether it is or whether it's more like a proxy. Anyway, point being, in _normal_ use, it doesn't actually modify the element itself. 

Whew, that was a long way of suggesting that you shorten:

via Polymer.dom => using the Polymer DOM API, which ensures ...